tests/app/views/contexts/test_hub_context.py

```diff
@@ -98,11 +99,61 @@ def test_get_context(schema, progress_store, answer_store, list_store, router):
 
     expected_context = {
         "title": "Choose another section to complete",
-        "description": "You must complete all sections in order to submit this survey",
+        "guidance": "You must complete all sections in order to submit this survey",
         "rows": [],
         "submit_button": "Continue",
     }
 
     assert expected_context == hub.get_context(
         survey_complete=False, enabled_section_ids=router.enabled_section_ids
     )
+
+
+def test_get_context_custom_guidance_incomplete(
+    progress_store, answer_store, list_store, router
+):
+    schema = load_schema_from_name("test_hub_and_spoke")
+    hub_context = HubContext(
+        language=None,
+        progress_store=progress_store,
+        list_store=list_store,
+        schema=schema,
+        answer_store=answer_store,
+        metadata={},
+    )
+
+    expected_context = {
+        "title": "Choose another section to complete",
+        "guidance": "Guidance displayed on hub when incomplete",
+        "rows": [],
+        "submit_button": "Continue",
+    }
+
+    assert expected_context == hub_context.get_context(
+        survey_complete=False, enabled_section_ids=router.enabled_section_ids
+    )
+
+
+def test_get_context_custom_guidance_complete(
```

Comment:
This method would be clearer if it was named `test_get_context_custom_content_complete`, as the title and submit button also change.